### DISCUSSION TITLE
Failed connection

### DISCUSSION THREAD

**Discussion by Colins2** [blueman-project/blueman#2174] [Q&A]
blueman:
BlueZ:
Distribution:
Desktop environment:

* [ ] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done my best effort to follow.

I have just installed Debian 12.2. I get the Bluetooth icon and I can go through the steps to connect to a Bluetooth speaker. It appears to connect and then drops the connection after about 5 seconds with a message:
"Connection Failed: br-connection-page-timeout"
The application was part of the distro, not something extra I have installed.
The speaker works fine and pairs instantly with my MacBook

**Comment by infirit** [comment]
Most likely you don't have PulseAudio installed. https://github.com/blueman-project/blueman/wiki/Troubleshooting#a2dp-audio-device-paired-but-fails-to-connect
